api/repositories/pod_repository_test.go

```diff
@@ -0,0 +1,112 @@
+package repositories_test
+
+import (
+	apierrors "code.cloudfoundry.org/korifi/api/errors"
+	"code.cloudfoundry.org/korifi/api/repositories"
+	korifiv1alpha1 "code.cloudfoundry.org/korifi/controllers/api/v1alpha1"
+	"code.cloudfoundry.org/korifi/tests/matchers"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("PodRepository", func() {
+	var (
+		podRepo  *repositories.PodRepo
+		org      *korifiv1alpha1.CFOrg
+		space    *korifiv1alpha1.CFSpace
+		pod      *corev1.Pod
+		appGUID  string
+		app      *repositories.AppRecord
```

Comment:
No need to use app record here, sicne `DeletePod` does not take AppRecord as an argument. We could just add a var for the app revision.

Reply:
done